admin/admin-service/src/main/java/io/mosip/admin/validator/DocCatCode.java

```diff
@@ -0,0 +1,23 @@
+package io.mosip.admin.validator;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Documented
+@Constraint(validatedBy = DocCatCodeValidator.class)
+@Target({ ElementType.FIELD, ElementType.TYPE_USE, ElementType.PARAMETER })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface DocCatCode {
+
+	String message() default "docCategory is Invalid";
```

Comment:
It is always better to set error_code in the message instead of message directly.